src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -97,9 +98,144 @@ def custom_REE_plant_currency_units():
+            "cost of ccapital (ATWACC). For the UKy case study, the ATWACC is"
+            "5.77%.",
+        ),
+    )
+    CONFIG.declare(
+        "plant_lifetime",
+        ConfigValue(
+            default=None,
+            domain=float,
+            description="length of operating period in years.",
+        ),
+    )
+    CONFIG.declare(
+        "total_capital_cost",
+        ConfigValue(
+            default=None,
+            description="value for total capital cost; ignored if no value is "
+            "passed.",
+        ),
+    )
+    CONFIG.declare(
+        "annual_operating_cost",
+        ConfigValue(
+            default=None,
+            description="value for total operating cost; ignored if no value "
+            "is passed.",
+        ),
+    )
+    CONFIG.declare(
+        "annual_revenue",
+        ConfigValue(
+            default=None,
+            description="value for total revenue; ignored if no value is " "passed.",
+        ),
+    )
+    CONFIG.declare(
+        "cost_year",
+        ConfigValue(
+            default=None,
+            domain=str,
+            description="assumed project start year for costs, which is the "
+            "basis for NPV results; ignored if no value is passed.",
+        ),
+    )
+    CONFIG.declare(
+        "has_capital_expenditure_period",
+        ConfigValue(
+            default=False,
+            domain=bool,
+            description="True/false flag whether a capital expenditure period "
+            "occurs.",
+        ),
+    )
+    CONFIG.declare(
+        "capital_expenditure_percentages",
+        ConfigValue(
+            default=None,
+            domain=list,
+            description="a list of values that sum to 100 representing how "
+            "capital costs are spread over a capital expenditure period; for "
+            "example, an input of [10, 60, 30] is parsed as a 3-year period "
+            "where capital costs are spread as 10% in year 1, 60% in year 2, "
+            "and 30% in year 3. The capital period precedes the operating "
+            "period, for example an input of [100] means that 100% of capital "
+            "expenses occur in the year preceding the operating period. Set "
+            "to None to indicate no expenditure period, which means that all "
+            "capital expenses occur in the first operating year t=0.",
+        ),
+    )
+    CONFIG.declare(
+        "capital_escalation_percentage",
+        ConfigValue(
+            default=3.6,
+            domain=float,
+            description="rate at which capital costs escalate during the "
+            "capital expenditure period. Set to 0 to indicate expenditure is "
+            "spread but there is no cost escalation.",
+        ),
+    )
+    CONFIG.declare(
+        "capital_loan_interest_percentage",
+        ConfigValue(
+            default=6,
+            domain=float,
+            description="interest rate for capital" "equipment loan repayment.",
```

Comment:
Remove the  " " from the middle of the doc string (this was caused by `black` removing an unneeded line break).